randomcode.py

```diff
@@ -0,0 +1,3 @@
+import random
```

Comment:
Nicely imported, my friend.

Reply:
Thanks! This comment feature looks like it will be real useful during collaboration.